src/react/src/util/util.js

```diff
@@ -893,6 +893,8 @@ export const convertFeatureFlagsObjectToListOfActiveFlags = featureFlags =>
 export const checkWhetherUserHasDashboardAccess = user =>
     get(user, 'is_superuser', false);
 
+export const claimFacilityIntroIsValid = ({ agreement }) => agreement;
```

Comment:
What is the purpose of this function, which returns a value without changing it?

Reply:
@Innavin369 We can use StubTrue instead of it.

Reply:
This value is boolean and changes by radio btn.